pkg/forwarder/transaction/transaction.go

```diff
@@ -384,3 +385,26 @@ func (t *HTTPTransaction) SerializeTo(serializer TransactionsSerializer) error {
 	log.Trace("The transaction is not stored on disk because `storableOnDisk` is false.")
 	return nil
 }
+
+// readResponseBody read HTTP response body. If body is more than 1000 bytes
+// truncates it to  prevent from logging a huge message. When EOF occurs,
```

Comment:
This seems like not a normal space

```suggestion
// truncates it to prevent from logging a huge message. When EOF occurs,
```